lib/tasks/manual_eligibility_demo.rake

```
@@ -0,0 +1,200 @@
+namespace :eligibility_demo do
+  def eligibility_demo_exam_title = "Prüfungszulassung Demo – Finale Zulassung"
```

RuboCop (1.88.2)

[fatal] unexpected token tEQL
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)